### PR TITLE
Improvement/data testid to elements in ui

### PR DESCRIPTION
**Pull request by jantocv**
## Description
Propagate `data-testid` prop to unit test with `preact-testing-library` or other framework by fetching by that identifier.

## Checks
- [ ] Requires documentation update
- [x] Requires lib version update

## Versioning impact
- [ ] **MAJOR** incompatible API changes.
- [x] **MINOR** adds functionality in a backwards-compatible manner.
- [ ] **PATCH** backwards-compatible bug fixes.

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
+{
```
Comment (rahulpsingh) [inline .vscode/settings.json]: This can be ignored!

Comment (jantocv) [reply .vscode/settings.json]: wops! sorry and thanks!

**Comment by JoseOrtiz** [comment]
Can you squash some commits?

**Comment by jantocv** [comment]
> Can you squash some commits?

Yes, I did small commits, however I am going to squash + merge to push to master in a single one

**Pull request merged**